core/designsystem/src/main/kotlin/com/dhc/designsystem/graph/DhcGraph.kt

```diff
@@ -0,0 +1,288 @@
+            data = graphData,
+            maxValue = dhcGraphConfig.maxValue,
+            modifier = Modifier
+                .padding(
+                    start = 16.dp,
+                    top = rowLabelStyle.fontSize.value.dp / 2,
+                    bottom = rowLabelStyle.fontSize.value.dp / 2 + columnLabelStyle.fontSize.value.dp,
+                )
+                .constrainAs(graphBars) {
+                    bottom.linkTo(graphRowLines.bottom)
+                    start.linkTo(graphRowLines.start)
+                    end.linkTo(graphRowLines.end)
+                    top.linkTo(graphRowLines.top)
+                    width = Dimension.fillToConstraints
+                    height = Dimension.fillToConstraints
+                }
+        )
+
+        DhcGraphColumnLabel(
+            modifier = Modifier
+                .padding(start = 16.dp)
+                .fillMaxWidth()
+                .constrainAs(graphColumnLabels) {
+                    bottom.linkTo(parent.bottom)
+                    start.linkTo(graphRowLines.start)
+                    end.linkTo(graphRowLines.end)
+                    width = Dimension.fillToConstraints
+                },
+            datas = graphData,
+        )
+    }
+}
+
+@Composable
+private fun DhcGraphRowLines(
+    lines: Int,
+    textHeight: Dp,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier,
+        verticalArrangement = Arrangement.SpaceBetween,
+    ) {
+        repeat(lines) {
+            Box(
+                modifier = Modifier.height(textHeight),
+                contentAlignment = Alignment.Center,
+            ) {
+                Box(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .height(1.dp)
+                        .background(SurfaceColor.neutral600)
+                )
+            }
+        }
+    }
+}
+
+@Composable
+internal fun DhcGraphRowLabels(
+    labels: List<String>,
+    labelStyle: TextStyle,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        modifier = modifier,
+        horizontalAlignment = Alignment.Start,
+        verticalArrangement = Arrangement.SpaceBetween,
+    ) {
+        labels.forEach { label ->
+            Text(
+                text = label,
+                color = SurfaceColor.neutral200,
+                style = labelStyle,
+            )
+        }
+    }
+}
+
+@Composable
+private fun DhcGraphBar(
+    color: Color,
+    modifier: Modifier = Modifier,
+) {
+    Box(
+        modifier = modifier
+            .clip(
+                shape = RoundedCornerShape(
+                    topStart = 8.dp,
+                    topEnd = 8.dp,
+                )
+            )
+            .background(color = color),
+    )
+}
+
+@Composable
+private fun DhcGraphBars(
+    data: List<DhcGraphData>,
+    maxValue: Int,
+    modifier: Modifier = Modifier,
+) {
+    val colors = LocalDhcColors.current
+    val graphWidth = 48.dp
```

Comment:
요것도 DhcGraphBars 의 parameter 로 받아오면 좋을지두?
```suggestion
@Composable
private fun DhcGraphBars(
    data: List<DhcGraphData>,
    maxValue: Int,
    modifier: Modifier = Modifier,
    graphWidth: Dp = 48.dp,
) {
    val colors = LocalDhcColors.current
```
뭔가,, Composable 함수 내에서 변수로 정의하여 가져오면, 
recomposable 이 발생할 때 마다 Composable 함수가 재실행되면서 graphWidth 를 위한 변수가 stack 에 계속 쌓일 것 같은데,,,
만약 paraemter 로 받아오면 그런 점을 덜 수 있어서 좋지 않을까란 생각이 드넹

그런면에서 DhcGraphBars 의 parameter 로 받아오는건 어떨까?